awx/ui_next/src/components/LaunchPrompt/steps/OtherPromptsStep.jsx

```diff
@@ -34,6 +34,16 @@ function OtherPromptsStep({ config, i18n }) {
           information and examples on patterns.`)}
         />
       )}
+      {config.ask_scm_branch_on_launch && (
+        <FormField
+          id="prompt-scm-branch"
+          name="scm-branch"
```

Comment:
I think the API expects this as `scm_branch` with an underscore

Reply:
You'll also need to update the `submit` function in `LaunchPrompt` to send this value through to the API

Reply:
Updated per comments. Thanks for your review @keithjgrant .